### PR TITLE
Check for double deposits on mini launchpad

### PR DESCRIPTION
**Pull request by pk910**
Check for double deposits on mini launchpad:
image

image

### CODE REVIEW

File: handlers/submit_deposit.go [hunk 1]
```diff
@@ -82,3 +101,73 @@ func buildSubmitDepositPageData() (*models.SubmitDepositPageData, time.Duration)
...
 	return pageData, 1 * time.Hour
 }
+
+func handleSubmitDepositPageDataAjax(w http.ResponseWriter, r *http.Request) error {
+	query := r.URL.Query()
+	var pageData interface{}
+
+	switch query.Get("ajax") {
+	case "load_deposits":
+		if r.Method != http.MethodPost {
```
Comment (skylenet) [inline handlers/submit_deposit.go:111]: Just a minor nit: In theory `/ajax=load_deposits`  should be a `GET` request cause no data is being changed.

**Review by skylenet — APPROVED** [review]

File: ui-package/src/components/SubmitDepositsForm/DepositEntry.tsx [hunk 2]
```diff
@@ -79,6 +87,109 @@ const DepositEntry = (props: IDepositEntryProps): React.ReactElement => {
...
+                      </div>
+                    </div>
+                  </div>
+                  <div className="d-flex">
+                    <div className="tx-details-label">
+                      Block:
+                    </div>
+                    <div className="tx-details-value">
+                      <div className="d-flex">
+                        <span className="flex-grow-1 text-truncate">{tx.block}</span>
```
Comment (skylenet) [inline ui-package/src/components/SubmitDepositsForm/DepositEntry.tsx:122]: Maybe a link to the block explorer and respective block number would be a nice to have.

Comment (skylenet) [reply ui-package/src/components/SubmitDepositsForm/DepositEntry.tsx:122]: Nevermind. I just saw you have a link to the TX if the explorerUrl exists.

File: handlers/submit_deposit.go (hunk 1 above)
Comment (pk910) [reply handlers/submit_deposit.go:111]: I'm not sure about this, `GET` data is limited by url size, so it's only suitable for limited amount of information.
In this situation I'm sending in a unbound list of public keys, each consuming 96Bytes in hex format.
When using the GET approach we'll hit url size limits at some point with large deposit files.
eg. 100 pubkeys will already consume 9600 bytes, leaving out any delimiter, argument name or other stuff that'S needed for valid urls ;)

Comment (skylenet) [reply handlers/submit_deposit.go:111]: Yep, that's a very valid reason to not use GET 👍  Makes sense to move that to the HTTP body.

**Pull request merged**